src/utils/FormMarshaler.js

```diff
@@ -57,7 +57,7 @@ export default class FormMarshaler extends ObjectSerializer {
       if (!payload.hf) {
         payload.hf = (new Array(3)).fill('');
         payload.hf_styles = ['uppercase', 'uppercase', 'uppercase'];
-        payload.hf_size = [3, 1, 1];
+        payload.hf_sizes = [3, 1, 1];
```

Comment:
@rsumilang we can't change core machinery of form processing/marshalling/etc. for specific fields.  gnarles barkley.  this needs to be contained within the component that provides the fields.  for example, creating some unmanaged fields and using the field registration with final form api and having a "beforeSubmit" that converts the value before form submission.

this feature must be self contained tho.